oaknational/Oak-Web-Application
# Feat/add hubspot to downloads

## Description

connect downloads submit button to hubspot api and form

## Issue(s)

Fixes #1576 

## How to test

1. Go to downloads page and Teacher Hub - Email Signup Modal form on the dev hubspot site
2. Click on download button
3. You should see school and email details sent to the hubspot form

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
